### ISSUE TITLE
Change how initial demo languages are loaded

### ISSUE THREAD

**Issue by jzohrab** [LuteOrg/lute-v3#534]
Currently, languages are pre-loaded into the startup database.  This is a hassle sometimes, as changing a language's stories (e.g the Tutorial) can cause the entire DB file to change, even though it's only data that's changed.  The data exists in the language def files, there's no need for that thrash to impact the db diffs as well.  This will also simplify database versioning, I think.

To do:

- [x] add a system flag, something like 'shouldLoadDemoData', which is only set for the new database
- [x] on app startup, check that flag, and load from the appropriate def files
- [x] once app is started, remove 'shouldLoadDemoData' flag
- [x] if any data exists, nothing should happen even if 'shouldLoadDemoData' flag is present - can't write over existing stuff
- [x] get all tests passing again
- [x] look into speeding up the language defs load -- can probably just load all of the yaml only and cache it in a dict
- [x] move all existing tests from stub file to the test_demo.py
- [x] add any remaining tests
- [x] remove stub test file test_issue_534 ... for db demo redesign
- [x] change the LoadDemoData flag, should only be present or deleted
- [x] add message, "loading sample data"
- [x] test release with new kind of empty db, try install and see if works
- [x] delete install, remove mecab, do install, see if a lang is marked as a preload but its parser isn't supported then we shouldn't do that
- [x] try running install with a copy of my own prod data
- [x] change db.reset to use new baseline stuff, remove old baseline stuff
- [x] remove db.reset inv task?
- [x] fix docs, have to re-baseline
- [x] change the db.demo functions to a service object, simplifies importing
- [x] remove empty.db, db_export_empty(c), def db_wipe(c), test for db wipe?, check for "wipe", "empty"
- [x] final beta release and fresh install?

**Comment by jzohrab** [comment]
Started work in `issue_534_change_initial_demo_data_load` branch.  Currently 127 pytest tests fail, as the baseline db no longer has the demo languages and books.  `pytest -x`and fix the rest.

**Comment by jzohrab** [comment]
re speeding up the tests: test perf chars:

```
this branch
unit: 393 passed in 115.20s (0:01:55)
accept: 45 passed in 209.42s (0:03:29)

develop
unit: 385 passed in 46.24s
accept: 45 passed in 210.53s (0:03:30)
```

so the unit tests are quite a bit slower.

**Comment by jzohrab** [comment]
All seems good, opening PR for CI check.

**Comment by jzohrab** [comment]
Released in 3.6.1.
